app/helpers/patient_query_helper.rb

```diff
@@ -628,6 +628,7 @@ def advanced_filter_common_exposure_cohorts(patients, filter)
     filter[:value].each do |field|
       # Map multi-select type filter from { label, value } to values
       value = field[:value].pluck(:value)
+      next unless value.present?
 
       # Get patients where cohort value is any of the cohort values specified in the filter
       case field[:name]
```

Comment:
I think line 664 needs to be updated, I'm seeing the number of monitorees on the page change even if I leave the field blank

Reply:
See here:
https://user-images.githubusercontent.com/35042815/143126194-29b5f29d-d223-4d5c-a17e-afc935d06a46.mov